src/Examine.Lucene/Suggest/SuggesterContext.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Collections.Generic;
+using Examine.Lucene.Indexing;
+using Lucene.Net.Index;
+using Lucene.Net.Search.Suggest;
+using Lucene.Net.Util;
+
+namespace Examine.Lucene.Suggest
+{
+    /// <summary>
+    /// Suggester Context for LuceneSuggester
+    /// </summary>
+    public class SuggesterContext : ISuggesterContext
+    {
+        private ReaderManager _readerManager;
+        private FieldValueTypeCollection _fieldValueTypeCollection;
+        private readonly SuggesterDefinitionCollection _suggesterDefinitions;
+        private readonly Dictionary<string, ILookupExecutor> _suggesters;
+
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="readerManager">Reader Manager for IndexReader on the Suggester Index</param>
+        /// <param name="fieldValueTypeCollection">Fields of Suggester Index</param>
+        public SuggesterContext(ReaderManager readerManager, FieldValueTypeCollection fieldValueTypeCollection, SuggesterDefinitionCollection suggesterDefinitions,
+            Dictionary<string, ILookupExecutor> suggesters)
+        {
+            _readerManager = readerManager;
+            _fieldValueTypeCollection = fieldValueTypeCollection;
+            _suggesterDefinitions = suggesterDefinitions;
+            _suggesters = suggesters;
+        }
+
+        /// <inheritdoc>/>
+        public IIndexReaderReference GetIndexReader() => new IndexReaderReference(_readerManager);
+
+        public SuggesterDefinitionCollection GetSuggesterDefinitions() => _suggesterDefinitions;
```

Comment:
This is public and should therefore have XML documentation - Use `/// <inheritdoc />` where appropriate

Reply:
Done